install.sh

```diff
@@ -214,6 +214,12 @@ get_iso()
                 break
             fi
         done
+    else
```

Comment:
This is not enough if there happen to have another disk mountable but actually contains no k3os, right?

Reply:
Not true. The else is fired when there actually is a k3os labeled partition. It just removes the number (/dev/sdb3 to /dev/sdb in my case), so the mounting further down the line works.